### PR TITLE
Search bar

### PR DESCRIPTION
**Pull request by omarawad0**
- used the TextInput component to create the search bar box UI.
- Currently, the search bar UI is disabled and available only for Desktop users.
- fixed a typo on the login page

### CODE REVIEW

File: modules/shared/components/molecules/Navigation/Navigation.module.css [hunk 1]
```diff
@@ -17,3 +17,7 @@
 .nav-links li.nav-user {
   @apply md:mr-0;
 }
+
+.search-box {
+  @apply hidden md:block ml-l w-19xl bg-grey-bg border border-grey-shd5 rounded-md;
+}
```
Comment (ahmedelsayad11) [inline modules/shared/components/molecules/Navigation/Navigation.module.css:23]: A small issue here, the search bar is not centered with the logo as it is in the design, you may need to adjust it a little.
this is the design
Capture
this is the pr
Capture

**Review by ahmedelsayad11 — CHANGES_REQUESTED** [review]
your pr might have some conflict when the navbar modification pr gets to merge.

File: modules/shared/components/molecules/Navigation/Navigation.module.css (hunk 1 above)
Comment (omarawad0) [reply modules/shared/components/molecules/Navigation/Navigation.module.css:23]: requested changes applied

**Review by ahmedelsayad11 — APPROVED** [review]
LGTM

**Pull request merged**